stoobly_agent/app/cli/helpers/openapi_endpoint_adapter.py

```diff
@@ -94,7 +107,7 @@ def adapt(self, spec: Spec) -> List[EndpointShowResponse]:
             elif parsed_url.scheme == 'http':
               endpoint['port'] = '80'
             else:
-              endpoint['port'] = ''
+              endpoint['port'] = '443'
```

Comment:
I gave it a default value since the 'port' parameter in the POST body had to be a non-empty value. Got a bad request error when I left it as `''`

Reply:
Since endpoints with the same path can belong to two different services, we do require `host` and `port`. This should be changed on API side so that `host` and `port` are optional and instead require a `service_name` value. Until that happens, when no `host` is provided, we can set it as `-` and when no port is provided, we can set it as `0`.